enclave-tls/dist/rpm/enclave_tls.spec

```diff
@@ -51,9 +54,12 @@ popd
 
 %install
 pushd %{name}
-Enclave_Tls_Root=%{?buildroot}/opt/enclave-tls Enclave_Tls_Bindir=%{?buildroot}/usr/share/enclave-tls/samples make install
+Enclave_Tls_Root=%{?buildroot}%{ENCLAVE_TLS_ROOTDIR} Enclave_Tls_Bindir=%{?buildroot}%{ENCLAVE_TLS_BINDIR} make install
 popd
 
+%postun
+rm -rf %{ENCLAVE_TLS_ROOTDIR} %{ENCLAVE_TLS_BINDIR}
```

Comment:
/usr/share/enclave-tls/ should be removed as well.

Reply:
OK